CSS-Grid-Opdracht/styles.css

```diff
@@ -1,3 +1,60 @@
-#test {
-    color: blue;
+body {
+    background-color: #ffefdb;
+}
+.class-items {
+    background-color: #ba3b44;
+    border: solid;
+    border-width: 2px;
+    border-color: #932f37;
+    padding: 16px;
+    color: white;
+}
+
+.container {
+    display: grid;
+    grid-gap: 16px 16px;
```

Comment:
Nice dat je het in een regel hebt verwerkt! Het had ook in een `column-wrap` en een `row-gap` gekund, maar dit is veel cleaner! 👍